specification/azsadmin/resource-manager/user-subscriptions/Microsoft.Subscriptions/preview/2015-11-01/Subscriptions.json

```diff
@@ -0,0 +1,412 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2015-11-01",
+        "title": "SubscriptionsManagementClient",
+        "description": "The User Subscriptions Management Client."
+    },
+    "host": "management.local.azurestack.external",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the list of subscriptions.": {
+                        "$ref": "./examples/Subscriptions/List.json"
+                    }
+                },
+                "description": "Get the list of subscriptions.",
+                "tags": [
+                    "Subscriptions"
+                ],
+                "operationId": "Subscriptions_List",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/SubscriptionList"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}": {
+            "get": {
+                "x-ms-examples": {
+                    "Get the specified subscription.": {
+                        "$ref": "./examples/Subscriptions/Get.json"
+                    }
+              },
+              "tags": [
+                "Subscriptions"
+              ],
+              "operationId": "Subscriptions_Get",
+              "description": "Gets details about particular subscription.",
+              "parameters": [
```

Comment:
Why are you not using the SubscriptionIdParameter here?

Reply:
Subscription is the main resource for this spec, I dont want to mask it in global param

Reply:
Then change it from a global parameter to a passed in parameter.  There is no need to repeat the same parameter in multiple places.